src/Cell.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class Cell extends React.Component{
+  constructor(props){
+    super(props);
+    this.startGame = this.startGame.bind(this);
+    this.getKeyDirection = this.getKeyDirection.bind(this);
+    this.foodPosition = 50;
+    this.state = { color : 'white', snakeHead: 0, direction: ''};
+  }
+  getKeyDirection(event){
+     switch(event.keyCode){
+        case 37:
+          this.setState({direction :'left',
+          snakeHead: this.state.snakeHead-1});
+          break;
+        case 38:
+          this.setState({direction :'up',});
+          break;
+        case 39:
+          this.setState({direction :'right',
+          snakeHead: this.state.snakeHead+1});
+          break;
+        case 40:
+          this.setState({direction :'down'});
+          break;
+      }
+  }
+
+  startGame(){
+    console.log(this.state,"This is my state in startgame");
+    window.setInterval(() =>{
```

Comment:
You would be creating a new setInterval everytime you fire startGame function.. Are you sure this is what you want. This is a major surce for memory leak 
without clearInterval